build.assets/Makefile

```diff
@@ -18,27 +18,32 @@ TAG ?=
 TELE_FLAGS :=
 CWD := $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
 
+TELE ?= $(shell command -v tele;)
```

Comment:
for CI/CD why don't you download tele of the right version to the work directory? may be you are already doing that, just missed it

Reply:
That's what this change is for. So on the Jenkins server I have it download tele and then pass the path as TELE=${WORKDIR}/tele but on my machine for testing I can just use the installed tele since it's new. 

That was my thought process anyway. I suppose I could move the download tele into the makefile instead of doing it on the jenkins side.

Reply:
ok, cool!